typings/externs.d.ts

```diff
@@ -57,15 +57,21 @@ declare global {
     export import CrdpEvents = _CrdpMappings.Events;
     export import CrdpCommands = _CrdpMappings.Commands;
 
+    /** Simulation settings that control the amount of network & cpu throttling in the run. */
```

Comment:
thx for adding all these! awesome.

We currently dont have a technique to keep these matching the proto ones, right?

Reply:
correct, I think @brendankenny mentions that we don't but should :cry: